requirements-dev.txt

```diff
@@ -1,4 +1,4 @@
-pywikibase == 0.0.2
+pywikibase >= 0.0.3
```

Comment:
You should set an upper-bound on this so that if you do some breaking changes to pywikibase and increment the minor version, we don't accidentally install that. 

E.g. `pywikibase >= 0.0.3, < 0.0.999`